internal/service/staticip/static_ip.go

```diff
@@ -1,4 +1,4 @@
-package static_ip
+package staticip
```

Comment:
dont know liked it better before, feel free to ignore me though

Reply:
so am I, but https://go.dev/blog/package-names

Reply:
currently, we don't have any specific code style guidelines in this repo, so we would wanna use general guidelines that exist in the Go community, just to make sure this is not confusing anyone who's going to work with this repo

anything that deviates from the common guidelines must follow general Go guidelines, until we add our own guidelines